ipynb/examples/android/benchmarks/Android_Geekbench.ipynb

```diff
@@ -0,0 +1,572 @@
+    "            data = {}\n",
+    "            for workload in workloads:\n",
+    "                data[workload] = {}\n",
+    "                for field in benchmark_fields:\n",
+    "                    data[workload][field] = self.benchmarks[benchmark][workload][field]        \n",
+    "            benches[benchmark] = data\n",
+    "            \n",
+    "        return benches"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "def display_bench_results(geekbench, detailed=False):\n",
+    "    print \"===== Global results =====\"\n",
+    "    \n",
+    "    scores = geekbench.global_scores()\n",
+    "    \n",
+    "    # Build dataframe for display\n",
+    "    row = []\n",
+    "    for bench_type, score in scores.iteritems():\n",
+    "        row.append(score)\n",
+    "        \n",
+    "    df = pd.DataFrame(data=row, index=scores.keys(), columns=[\"Global score\"])\n",
+    "    display(df)\n",
+    "    \n",
+    "    if not detailed:\n",
+    "        return\n",
+    "    \n",
+    "    print \"===== Detailed results =====\"\n",
+    "    \n",
+    "    scores = geekbench.detailed_scores()\n",
+    "    \n",
+    "    for benchmark, results in geekbench.detailed_scores().iteritems():\n",
+    "        print \"----- {} benchmark -----\".format(benchmark)\n",
+    "        # Build dataframe for display\n",
+    "        data = []\n",
+    "        idx = []\n",
+    "        columns = results.values()[0].keys()\n",
+    "        for workload, fields in results.iteritems():\n",
+    "            data.append(tuple(fields.values()))\n",
+    "            idx.append(workload)\n",
+    "        display (pd.DataFrame(data=data, index=idx, columns=columns))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true,
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "for f in os.listdir(te.res_dir):\n",
+    "    if f.endswith(\".gb4\"):\n",
+    "        geekbench = Geekbench(te.res_dir + \"/\" + f)\n",
+    "        \n",
+    "        print \"Analysing geekbench {}\".format(geekbench.name())\n",
+    "        display_bench_results(geekbench, False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Analysing several runs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "It can be interesting to compare Geekbench results with different parameters (kernel, drivers) and even different devices to gauge the impact of these parameters. As Geekbench results can vary a bit from one run to another, having a set of repeated runs is preferable.\n",
```

Comment:
Yes, you're absolutely right. I ended up having around a hundred runs to compare, and wanted to add that into the example notebook, but it is probably overkill for a simple example. I've modified it so it simply pulls stats from the Geekbench_example_* directories - this functionality can be used in another notebook to compare runs on different devices.